src/conic.jl

```diff
@@ -879,7 +912,7 @@ function MathProgBase.optimize!(m::PajaritoConicModel)
 
     # Release the inf_conic_model if applicable
     if applicable(MathProgBase.freemodel!,ini_conic_model)
-        MathProgBase.freemodel!(ini_conic_model)
+        !contains(string(typeof(m.cont_solver)), "MosekSolver") && MathProgBase.freemodel!(ini_conic_model)
```

Comment:
Hmmm I am still getting freemodel crash from Mosek (I have your fix under git log of Mosek)